app/views/layouts/application.html.erb

```diff
@@ -35,7 +35,7 @@
       js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.3&appId=259908887444831";
       fjs.parentNode.insertBefore(js, fjs);
     }(document, 'script', 'facebook-jssdk'));</script>
-    <%= external_javascript_include_tag "jquery", "2.1.4" %>
```

Comment:
Was backwards compatibility broken between 2.x and 3.x? Could this cause issues elsewhere in the frontend?

Reply:
as people have pointed out there isnt alot of JS on the site to begin with and everything i wrote is compatible. the tables being used the way they currently are were not supported by jquery 2. ill give this another pass on the way by. but the very little js on the site worked fine on pass 1.

Reply:
everything seems to be in tip top shape and working as i would expect.